homework/module_1-hw_standardlib/task_2.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+
+def check_ip(ip_address) -> bool:
```

Comment:
I would simplify this function and it's hard to read and understand it

Reply:
fixed